pyeudiw/satosa/default/openid4vp_backend.py

```diff
@@ -342,6 +340,19 @@ def db_engine(self) -> DBEngine:
 
         return self._db_engine
 
+    def _build_authz_request_url(self, payload: dict) -> str:
+        scheme = self.config["authorization"]["url_scheme"]
+        if "://" not in scheme:
+            scheme = scheme + "://"
+        if not scheme.endswith("/"):
+            scheme = scheme + "/"
```

Comment:
not sure about this

haip:///?

while haip:// would be good

there is no requirements in having https urls with pending slashes, like https://example.org/